src/components/ImageSlider/ImageSlider.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Image, Box, IconButton, BoxProps } from '@chakra-ui/react';
+import { CloseIcon } from '@chakra-ui/icons';
+
+export interface ImagesInterface {
+  id: string;
```

Comment:
Id seems unnecessary since image url is always unique and can be used as ID

Reply:
@Harkunwar the reason i added id was cause i thought if the user uploads the same image twice (mistakenly), having a unique id per image allowed us to remove any instance of the (same) image without a problem. if i use the image url as the identifier, if we happen to upload the same image twice and then want to remove one, we can remove the first instance but if we want to also remove the others, we cannot do that as the image with that id has supposingly been removed previously from the array in handleFileRemoved(). what do you think? should we deal with this case in a more different way?

Reply:
@labcolab @gokcedilek Can this problem be solved if we never allow users to upload the same image twice? For example, if the user selects the same image twice, we only upload one image?